core/chains/evm/gas/models.go

```diff
@@ -81,6 +81,16 @@ func NewEstimator(lggr logger.Logger, ethClient feeEstimatorClient, chaintype ch
 	df := geCfg.EIP1559DynamicFees()
 
 	// create l1Oracle only if it is supported for the chain
+	// FIXME conflict, to be verified
```

Comment:
@matYang @amit-momin hey folks, you are the ones who might have a context on that, could you please help resolve that conflict? It seems that the code in the chainlink repository is completely different from what we have in ccip

Reply:
Cool, removing then